frontend/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "watch": "chokidar \"src/**/*\" \"next.config.js\" --initial --command=\"npm run watch_build\"",
+    "watch_build": "NODE_ENV='watch_build' npm run build",
+    "build": "next build && next export",
+    "start": "next start",
+    "lint": "next lint",
+    "test": "jest",
```

Comment:
It's run in CI, if that's what you mean? But it's not being stored anywhere; I'm not sure where to store that data, but that sounds like something useful to do indeed (and for the back-end as well).

Reply:
https://app.codecov.io/gh/firefox-devtools/profiler

I found this useful when we were ramping up our test coverage to track the overall trend. The graph is probably less interesting since the coverage has been staying fairly steady.